src/script/api/script_object.hpp

```diff
@@ -164,18 +164,28 @@ friend class TestScriptController;
 	 */
 	template <Commands Tcmd, typename Tret, typename... Targs>
 	struct ScriptDoCommandHelper<Tcmd, Tret(*)(DoCommandFlags, Targs...)> {
+		static bool DoMsg(StringID msg, Script_SuspendCallbackProc *callback, Targs... args)
+		{
+			return Execute(msg, callback, std::forward_as_tuple(args...));
+		}
+
+		static bool DoMsg(StringID msg, Targs... args)
+		{
+			return Execute(msg, nullptr, std::forward_as_tuple(args...));
+		}
+
 		static bool Do(Script_SuspendCallbackProc *callback, Targs... args)
 		{
-			return Execute(callback, std::forward_as_tuple(args...));
+			return Execute((StringID)0, callback, std::forward_as_tuple(args...));
```

Comment:
`STR_NULL` if possible, otherwise maybe `StringID{0}`?

Reply:
`STR_NULL` is not available in this header, and I don't want to include `table/strings.h` in a header just for that.
Maybe `{}` can be enough too.